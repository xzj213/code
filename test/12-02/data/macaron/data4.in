5
2134 176 2231 2771 1
2168 3304 3211 1009 3921
146 442 2673 2505 1785
438 423 1416 3677 2531
2544 1936 3374 814 3379
