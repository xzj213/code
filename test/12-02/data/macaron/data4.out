27 62 129 173 207 275 308 309 378 384 470 471 497 539 573 583 612 678 869 871 936 1024 1026 1102 1211 1215 1220 1249 1291 1426 1447 1461 1484 1486 1493 1505 1512 1533 1545 1610 1636 1656 1657 1671 1705 1800 1822 1856 1909 1955 1956 2012 2033 2057 2112 2138 2164 2199 2208 2233 2267 2294 2336 2339 2392 2423 2457 2488 2490 2496 2517 2640 2654 2717 2736 2781 2799 2919 2920 2969 2986 3000 3013 3049 3072 3161 3162 3258 3286 3348 3385 3394 3439 3461 3489 3584 3602 3634 3729 3769 3792 3794 3848 3910 3941 4005 4007 4008 4010 4039 4075 4091 4095 
